<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1054</width>
    <height>601</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Pneumonia Detector</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/images/play.svg</normaloff>
         <activeon>:/images/pause.svg</activeon>:/images/play.svg</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/images/stop.svg</normaloff>:/images/stop.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Save to csv</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Всего:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="countLabel">
       <property name="text">
        <string>0 шт.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ImagesTableWidget" name="tableWidget">
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>Название файла</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>normal (%)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>pneumonia (%)</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImagesTableWidget</class>
   <extends>QTableWidget</extends>
   <header>gui.widgets.images_tw</header>
   <slots>
    <signal>rows_removing_started()</signal>
    <signal>rows_removed(PyQt_PyObject)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>startButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWidget</receiver>
   <slot>on_start_required()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWidget</receiver>
   <slot>on_stop_required()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWidget</receiver>
   <slot>on_save_required()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>745</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableWidget</sender>
   <signal>rows_removing_started()</signal>
   <receiver>MainWidget</receiver>
   <slot>on_images_removing_started()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>526</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel">
     <x>526</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableWidget</sender>
   <signal>rows_removed(PyQt_PyObject)</signal>
   <receiver>MainWidget</receiver>
   <slot>on_images_removed(PyQt_PyObject)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>526</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel">
     <x>526</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_start_required(bool)</slot>
  <slot>on_stop_required()</slot>
  <slot>on_save_required()</slot>
  <slot>on_images_removed(PyQt_PyObject)</slot>
  <slot>on_images_removing_started()</slot>
  <slot>on_open_images_required()</slot>
 </slots>
</ui>
